ZenPacks/zenoss/LinuxMonitor/FileSystem.py

```diff
@@ -155,7 +132,12 @@ def getStorageServer(self):
                     if ':' in storage_device:
                         search_keywords.add('has-nfs-client:{}'.format(storage_device))
 
-                search_root = self.getDmdRoot('Devices').Storage
+                # Check if we have a /Storage device class yet
+                try:
+                    search_root = self.getDmdRoot('Devices').Storage
+                except Exception:
+                    pass
```

Comment:
This happend when there were no Storage Zp's installed.. Should there be any attempt to add in the /Storage dc?